Code-4-Community/hands-across-the-sea-frontend
# Gt.school and book infoin report

## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/866744728/pulses/1500401992)



Displays the current number of students and grades attended in the report by default (pulling this information from the previous report if there was one. Also displays the total number of books and the most recent shipment year from the school's book logs data. 

## This PR



Pretty straightforward, just getting this information from redux. One thing to note is that the most recent shipment year has to be the most recent shipment year from a book log that had a positive number of books, which is why I filtered it out before adding the shipment year.

This PR is also blocked by [this ticket](https://code4community-team.monday.com/boards/866744728/pulses/1543664620) since for cases where a school has no book logs, the shipment year will be null and this will get a bad request when you try to submit the form. So once that's merged and deployed I'll be able to merge this PR 

## Screenshots

 

<img width="995" alt="Screen Shot 2021-08-04 at 8 56 05 AM" src="https://user-images.githubusercontent.com/36246323/128184479-b584ad69-800f-430d-a007-fd5a6cc5f931.png">


## Verification Steps

 

Npm start, go to "Fill out a form". Pick a school that already has a report and go through the form and the book/grade values should show up. After that you can go to the book logs tab, add a book log or two for that school, and then go through the form again and then those values will show up.
